common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1059,6 +1060,11 @@ object PageElement {
           }
 
           case Some(interactive: InteractiveAtom) => {
+            val maybeRole = for {
+              contentAtom <- element.contentAtomTypeData
+              role <- contentAtom.role
+            } yield role
```

Comment:
I think this'd be better being added to the `Atom` model in frontend rather than taking it from the CAPI model separately

Reply:
Jon, see Chat discussion :)